ynput/OpenPype
# Maya: Fix extract review and thumbnail for Maya 2020

## Changelog Description

Fix playblasting in Maya 2020 with override viewport options enabled. Fixes #4730.

## Additional info

More info see issue #4730

I chose to adjust vendorized `capture.py` instead of editing both of maya's ExtractPlayblast and ExtractThumbnail plug-ins because I noticed it was already changed locally from the actual upstream capture.py - the Py3/2 compatibility was added differently upstream in the original capture.py repository. By doing this it also resolves issues with other usages of capture that maybe a studio had implemented.

## Testing notes:

1. Have override viewport options enabled in project settings (default)
2. Start Maya 2020
3. Publish review + thumbnail

Both should work fine.
